### DISCUSSION TITLE
Simple media app guide for a newbie

### DISCUSSION THREAD

**Discussion by dsjstc** [google/horologist#1259] [Q&A] [answered]
I'd like to follow [this Simple Media Player guide](https://google.github.io/horologist/simple-media-app-guide/), but it doesn't trivially build.

My last android project was before Compose, so I don't really know what I'm looking at... but I think the PlayerScreen instructions need to be updated.  I can't make it compile with a relatively recent install of Electric Eel, with what I think are the current Compose dependencies.

Can someone please link me to a version of this project updated to use current libraries, or with a functional build.gradle pointing to the old ones that worked when the page was published?

**Comment by yschimke** [comment]
As a starting point, try opening the Horologist project in Android Studio Flamingo. You should be able to immediately run the media/sample app.

If there are problems with the examples compiling can you post them in a bug and we can fix. Thanks for flagging this.

**Comment by luizgrp** [comment]
> but it doesn't trivially build.

Could you please paste the text or a screenshot of the error you are seeing?

**Reply by luizgrp** [reply]
fyi we've made some improvements to the guide and fixed some issues with the code

**Reply by dsjstc** [reply, marked as answer]
Turns out my Studio install was badly broken.  This recipe is working fine on Eel after a reinstall.  

As long as I'm here though, here are two minor points that could smooth the path for future newbies:

- Guide says add media-ui dependency to *project* build.grade.  I could only make it work at the bottom of the *app* build.gradle.
- Guide uses undefined gradle variables $horologist_version and $media3_version.  I defined them in gradle.properties, but maybe for simplicity the guide should just hardcode current-as-of-now version numbers.

Thanks again, this is awesome!

**Reply by luizgrp** [reply]
Thanks for confirming! 
For future reference, this [repo](https://github.com/luizgrp/HorologistSimpleMediaAppGuide) has the code of that guide in a wear app, ready to be run.
The first part of the guide is in the `main` branch, the second part is in the [functional](https://github.com/luizgrp/HorologistSimpleMediaAppGuide/tree/functional) branch.
